core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitor.java

```diff
@@ -347,18 +349,23 @@ private String getPeerTableName() {
       listenAddress = new InetSocketAddress(listenInetAddress, listenPort);
     }
 
-    return DefaultNodeInfo.builder()
-        .withEndPoint(endPoint)
-        .withBroadcastRpcAddress(broadcastRpcAddress)
-        .withBroadcastAddress(broadcastAddress)
-        .withListenAddress(listenAddress)
-        .withDatacenter(row.getString("data_center"))
-        .withRack(row.getString("rack"))
-        .withCassandraVersion(row.getString("release_version"))
-        .withTokens(row.getSetOfString("tokens"))
-        .withPartitioner(row.getString("partitioner"))
-        .withHostId(Objects.requireNonNull(row.getUuid("host_id")))
-        .withSchemaVersion(row.getUuid("schema_version"));
+    DefaultNodeInfo.Builder rv =
```

Comment:
Nit: we don't have rules for short-lived local variable names, but why `rv`?

Reply:
rv = return value.  I use it quite a bit in cases like this, although my preference would be to avoid the variable declaration all together and just return the result of the expression. That's pretty hard to do with a conditional application such as this, though.